syntax = "proto3";

package helloworld;

service HelloService {
    rpc SayHelloWorld (HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string message = 1;
}
